src/types/HeaderTypes.ts

```diff
@@ -1,11 +1,5 @@
-// TODO - share with banner/epic!
 import { OphanComponentType } from './OphanTypes';
-
-type Audience =
-    | 'AllExistingSupporters'
-    | 'AllNonSupporters'
-    | 'Everyone'
-    | 'PostAskPauseSingleContributors';
+import { Audience, Test, Variant } from './shared';
```

Comment:
I guess we can import `cta` too?

Reply:
annoyingly I called a field `url` and not `baseUrl`, so it'd be a breaking change!